oper8/watch_manager/python_watch_manager/utils/log_handler.py

```diff
@@ -14,14 +13,17 @@
 
 log = alog.use_channel("LOG-HANDLER")
 
+# Forward declaration of a queue for any type
+QUEUE_TYPE = "Queue[Any]"
+
 
 class LogQueueHandler(QueueHandler):
     """
     Log Handler class to collect messages from a child processes and pass
     them to the root process via a multiprocess queue
     """
 
-    def __init__(self, queue: "Queue[Any]", manifest: ManagedObject = None):
+    def __init__(self, queue: QUEUE_TYPE, manifest: ManagedObject = None):
```

Comment:
I'm not sure why I didn't reference Queue directly here. Is there a circular import somehow with multiprocessing?